build.js

```diff
@@ -58,7 +58,7 @@ async function main() {
       // Adds source mapping
       sourcemap: true,
       // The compiled code should be usable in node v10
-      target: 'node10',
+      target: 'node12',
```

Comment:
```suggestion
      target: 'node12.22',
```

This should be tested, but seems that it would work. Otherwise we can do es2019 which is the last version on which all features are supported by Node 12.22.0 (as per https://node.green )